src/commons/sagas/WorkspaceSaga.ts

```diff
@@ -1128,7 +1152,7 @@ export function* evalCode(
   }
 
   // For EVAL_EDITOR and EVAL_REPL, we send notification to workspace that a program has been evaluated
-  if (actionType === EVAL_EDITOR || actionType === EVAL_REPL) {
+  if (actionType === EVAL_EDITOR || actionType === EVAL_REPL || actionType === DEBUG_RESUME) {
```

Comment:
I'm not sure if this is the intended behaviour, but without this when in interpreter and ec-evaluator modes tabs do not spawn